### ISSUE TITLE
Access to Mozilla legal counsel

### ISSUE THREAD

**Issue by aDotInTheVoid** [rust-lang/foundation-faq-2020#50]
Will the rust foundation continue to have [access to mozilla legal counsel](https://forge.rust-lang.org/core/legal.html), or what will replace it?

**Comment by ashleygwilliams** [comment]
No, the Foundation will retain its own counsel, separate from Mozilla’s, as it will be an independent legal entity.
